src/Html/Abstractions/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+repo_root="$DIR/../.."
```

Comment:
If the file stays here
```suggestion
repo_root="$DIR/../../.."
```